tests/Tests/Unit/Transport/RestTransportTest.php

```diff
@@ -318,7 +319,6 @@ public function testStartServerStreamingCallThrowsRequestException()
 
         $this->getTransport($httpHandler, $apiEndpoint)
             ->startServerStreamingCall($this->call, []);
-        
```

Comment:
Probably you don't want to add this line.

Reply:
Same here... looks like most of the changes in this file are from a style checker